alby-nostr-wallet-connect/umbrel-app.yml

```
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: alby-nostr-wallet-connect
+name: Nostr Wallet Connect
+tagline: The power of the zaps at the tip of your fingers.
+icon: https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_icon.svg
+category: Finance
```

we just updated our categories yesterday. can you please change to all lowercase `finance`.

```suggestion
category: finance
```